prow/cmd/build/controller.go

```diff
@@ -17,30 +17,23 @@ limitations under the License.
 package main
```

Comment:
Just for my understanding, why do we keep anything of this controller?

Reply:
i'm not sure if it is completely knative/build specific, isn't it the prow-pipeline itself ?

Reply:
@fejta would be the authoritative source to confirm this but I believe so. The pipeline controller is in `prow/cmd/pipeline`

Reply:
You're rignt, we can remove it.